components/clp-package-utils/clp_package_utils/scripts/compress.py

```diff
@@ -21,28 +25,165 @@
 logger = logging.getLogger(__file__)
 
 
-def main(argv):
-    clp_home = get_clp_home()
-    default_config_file_path = clp_home / CLP_DEFAULT_CONFIG_FILE_RELATIVE_PATH
+def _generate_targets_list(
+    container_targets_list_path: pathlib.Path,
+    parsed_args: argparse.Namespace,
+) -> None:
+    input_type = parsed_args.input_type
+
+    if InputType.FS == input_type:
+        compression_targets_list_file = parsed_args.path_list
```

Comment:
How about `host_logs_list_path`? `host_` to differentiate it from `container_` and `path` to indicate its a path rather than an open file object.